core/src/main/scala/neotypes/types.scala

```diff
@@ -1,7 +1,5 @@
 package neotypes
 
-package object types {
-
-  class Path[NODE, RELATIONSHIP](val nodes: Seq[NODE], val relationships: Seq[RELATIONSHIP], path: org.neo4j.driver.v1.types.Path)
-
+object types {
+  final class Path[NODE, RELATIONSHIP](val nodes: Seq[NODE], val relationships: Seq[RELATIONSHIP], path: org.neo4j.driver.v1.types.Path)
```

Comment:
Any reason for this not being a case class?

Reply:
It looks like I missed `case` 😁